tests/ludwig/schema/test_validate_config_misc.py

```diff
@@ -390,3 +392,25 @@ def test_ludwig_schema_serialization(model_type):
         raise TypeError(
             f"Ludwig schema of type `{model_type}` cannot be represented by valid JSON. See further details: {e}"
         )
+
+
+def test_encoder_descriptions():
+    schema = get_input_feature_jsonschema(MODEL_ECD)
+
+    for feature_schema in schema["items"]["allOf"]:
+        type_data = feature_schema["then"]["properties"]["encoder"]["properties"]["type"]
+        assert set(type_data["enumDescriptions"].keys()).issubset(set(type_data["enum"]))
```

Comment:
Also assert that type_data["enumDescriptions"].keys() is nonempty, since an empty set would trivially pass the `issubset` check

Reply:
Done!